clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -38,6 +38,9 @@ public WrapperSerde(Serializer<T> serializer, Deserializer<T> deserializer) {
 
         @Override
         public void configure(Map<String, ?> configs, boolean isKey) {
+            if (configs == null) {
```

Comment:
Not sure if we should add this -- a wrapper (de)serializer should do this check is appropriate -- also, a (de)serializer might need to do some initialization that does not depend on the config. Thus, we should forward the call and just pass through `null` IMHO.